### ISSUE TITLE
Training precision always 0

### ISSUE THREAD

**Issue by HuanJiML** [ZhaoJ9014/face.evoLVe.PyTorch#71]
I am trying to train IR_50 on MS-Celeb-1M/Align_112x112 on 4 GPUs using the author's default parameters. After 6 epochs, LFW accuracy reaches 95.77%.  But the training script reports always top1 and top5 accuracy 0:
```
Training Prec@1 0.000 (0.000)   Training Prec@5 0.000 (0.000)
```

@ZhaoJ9014 what could be reason for this behavior?
<img width="361" alt="Screenshot 2019-07-04 at 11 14 36" src="https://user-images.githubusercontent.com/52528990/60654834-fe243680-9e4c-11e9-90c7-62e9e2c05fba.png">
<img width="352" alt="Screenshot 2019-07-04 at 11 14 44" src="https://user-images.githubusercontent.com/52528990/60654838-011f2700-9e4d-11e9-9209-ed843c6eee08.png">

My training loss also looks really bad compared to this repo's tensorflow log picture: https://github.com/ZhaoJ9014/face.evoLVe.PyTorch/blob/master/disp/Fig13.png

**Comment by ZhaoJ9014** [comment]
> I am trying to train IR_50 on MS-Celeb-1M/Align_112x112 on 4 GPUs using the author's default parameters. After 6 epochs, LFW accuracy reaches 95.77%. But the training script reports always top1 and top5 accuracy 0:
> 
> ```
> Training Prec@1 0.000 (0.000)   Training Prec@5 0.000 (0.000)
> ```
> 
> @ZhaoJ9014 what could be reason for this behavior?
> <img alt="Screenshot 2019-07-04 at 11 14 36" width="361" src="https://user-images.githubusercontent.com/52528990/60654834-fe243680-9e4c-11e9-90c7-62e9e2c05fba.png">
> <img alt="Screenshot 2019-07-04 at 11 14 44" width="352" src="https://user-images.githubusercontent.com/52528990/60654838-011f2700-9e4d-11e9-9209-ed843c6eee08.png">
> 
> My training loss also looks really bad compared to this repo's tensorflow log picture: https://github.com/ZhaoJ9014/face.evoLVe.PyTorch/blob/master/disp/Fig13.png

Hi, I think this is normal for margin based loss function since the first several epochs are for warming up. You can train more epochs and see. As you can observed, we have trained the network for 120 epochs in total.
